docker-compose.yaml

```diff
@@ -91,8 +91,10 @@ services:
         environment:
             RABBITMQ_DEFAULT_USER: ${RABBITMQ_DEFAULT_USER}
             RABBITMQ_DEFAULT_PASS: ${RABBITMQ_DEFAULT_PASS}
+        volumes:
+            - ./other/rabbitmq/enabled_plugins:/etc/rabbitmq/enabled_plugins
     postgres:
-        image: timescale/timescaledb:1.1.0-pg10
+        image: postgres:latest
```

Comment:
dangerous to use the latest tag always. Please use the latest stable version of postgres.